### PR TITLE
Upgrade to Woodwork 0.8.1

### PR DESCRIPTION
**Pull request by chukarsten**
Addresses: #2772

### CODE REVIEW

File: evalml/pipelines/utils.py [hunk 1]
```diff
@@ -130,7 +131,9 @@ def _get_preprocessing_components(
 
     # The URL and EmailAddress Featurizers will create categorical columns
     categorical_cols = list(
-        X.ww.select(["category", "URL", "EmailAddress"], return_schema=True).columns
+        X.ww.select(
+            ["category", "URL", "EmailAddress", "IPAddress"], return_schema=True
```
Comment (freddyaboulton) [inline evalml/pipelines/utils.py:135]: @chukarsten How come we need to make this change? AutoMLSearch and our pipelines can't handle the `IPAddress` logical type yet.

File: evalml/tests/pipeline_tests/test_pipeline_utils.py [hunk 2]
```diff
@@ -83,6 +83,16 @@ def _get_test_data_from_configuration(
                     "https://github.com/alteryx/featuretools",
                 ]
                 * 2,
```
Comment (chukarsten) [inline evalml/tests/pipeline_tests/test_pipeline_utils.py:85]: I'm just going to leave these as they don't hurt.

**Review by freddyaboulton — APPROVED** [review]
@chukarsten Looks good! We can fix `build_conda_pkg` by upping the max ww version after you merge! Thank you!!

**Review by angela97lin — APPROVED** [review]
LGTM, thank you!! 😁

**Pull request merged**